src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataTest.java

```diff
@@ -887,6 +920,72 @@ public void testGetImportedKeysDW() throws SQLException {
         }
     }
 
+    @Test
+    // Validates the metadata data types defined by JDBC spec
+    // https://docs.oracle.com/javase/8/docs/api/java/sql/DatabaseMetaData.html#getColumns-java.lang.String-java.lang.String-java.lang.String-java.lang.String-
+    public void testValidateColumnMetadata() throws SQLException {
```

Comment:
Style. Please use JavaDoc comments and @Test should be after comment/before header.

Reply:
Done